etc/telegraf.conf

```diff
@@ -1631,6 +1631,23 @@
 #   ## Export metric collection time.
 #   # export_timestamp = false
 
+# # Configuration for the RedisTimeSeries server to send metrics to
```

Comment:
For future reference the telegraf.conf is automatically updated but thank you for covering your bases!